docs/2023/Sunbird ED/content detail page as widget/index.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/Sunbird ED/content detail page as widget
+---
+
+
+# Author
+Shivam Chaturvedi
+
+# Contact Info
+- Email: chaturvediji2306@gmail.com 
+- Linkedin: https://www.linkedin.com/in/shivam-chaturvedi-035995206/ 
+- Github: https://github.com/shivam-2306
```

Comment:
```
- [Email](chaturvediji2306@gmail.com)
- [Linkedin](https://www.linkedin.com/in/shivam-chaturvedi-035995206/)
- [Github](https://github.com/shivam-2306)
```

Reply:
Please also add mailto:
- [Email](mailto:chaturvediji2306@gmail.com)

Reply:
> Please also add mailto: `- [Email](mailto:chaturvediji2306@gmail.com)` @wasup-yash

Right shruti, I think @shivam-2306 needs to update this too